src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -63,14 +66,21 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
   const firestoreVenue = useSelector(currentVenueSelectorData);
   const venueToUse = venue ? venue : firestoreVenue;
 
+  const parentVenueId = venueToUse?.parentId;
+  const parentVenue = useSelector(parentVenueSelector);
+
+  const backToParentVenue = useCallback(() => {
+    if (!parentVenueId) return;
+
+    window.location.href = venueInsideUrl(parentVenueId);
```

Comment:
[NB] We should switch to using the react-router navigation helpers more than these explicit window.location.href things.

Is there any reason we can't at least use something from `utils/url` here?

Reply:
changed in https://github.com/sparkletown/sparkle/pull/1217/commits/002f1b4686eae7db7a910850d16dde1407bc4d2c